### PR TITLE
Papis bump

### PR DESCRIPTION
**Pull request by teto**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -2635,7 +2635,7 @@ in {
 
   paperwork-backend = callPackage ../applications/office/paperwork/backend.nix { };
 
-  papis = callPackage ../development/python-modules/papis { };
+  papis = disabledIf (!isPy3k) (callPackage ../development/python-modules/papis { });
```
Comment (jonringer) [inline pkgs/top-level/python-packages.nix]: please move this into the expression
```suggestion
  papis =callPackage ../development/python-modules/papis { };
```

File: pkgs/development/python-modules/python-doi/default.nix [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub }:
+buildPythonPackage rec {
+  pname = "python-doi";
+  version = "0.1.1";
```
Comment (jonringer) [inline pkgs/development/python-modules/python-doi/default.nix]: put disabled here
```suggestion
  version = "0.1.1";
  disabled = !isPy3k;
```

File: pkgs/development/python-modules/python-doi/default.nix [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub }:
```
Comment (jonringer) [inline pkgs/development/python-modules/python-doi/default.nix]: space  please :)
```suggestion
{ stdenv, buildPythonPackage, fetchFromGitHub }:

```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS (python38Packages.pytest broken on master)
diff LGTM
commit history LGTM
executable seems to work

```
https://github.com/NixOS/nixpkgs/pull/71756
1 package failed to build:
python38Packages.papis

3 package were build:
papis python37Packages.python-doi python38Packages.python-doi
```

**Comment by teto** [comment]
thanks for the review.

**Pull request merged**